web/cookiesCTF/main.py

```diff
@@ -0,0 +1,122 @@
+from bottle import route, run, request, response, static_file, get, template
+
+
+@route('/')
+def hello():
+	f = open('./Pages/index.html')
+	return f.read()
```

Comment:
If we can't use `static_file`, here and on let's use context managers instead:
```suggestion
	with open('./Pages/index.html') as f:
		return f.read()
```